csi/volumeutils.py

```diff
@@ -64,6 +68,80 @@ def get(self):
         """Get Volume name"""
         return self.volname
 
+def connectSocket(client_socket):
+    retry_interval = 60
+    max_retries = 5
+
+    retry_count = 0
+    connected = False
+
+    while retry_count < max_retries and not connected:
```

Comment:
This will not run if max_retries is exceeded. The program should then fail, rather than continue